.github/workflows/main.yml

```
@@ -159,6 +159,8 @@ jobs:
     uses: ./.github/workflows/documentation.yml
     with:
       version: ${{ needs.version.outputs.nextVersionTag }}
+      skipNxCache: ${{ vars.NX_SKIP_NX_CACHE == 'true' || github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release') }}
```

```suggestion
      skipNxCache: ${{ vars.NX_SKIP_NX_CACHE == 'true' }}
```
it only runs on push on main